src/app/containers/BulletedList/index.jsx

```diff
@@ -25,6 +45,8 @@ const BulletedListContainer = ({ blocks, className, ...rest }) => {
         script={script}
         service={service}
         dir={dir}
+        ref={linkBlock ? viewRef : null}
+        onClick={clickTrackerRef}
```

Comment:
Does this trigger if someone clicks anywhere on the list, not necessarily on a link?  Does it trigger when the list has no links?

Reply:
Yes to both. It's triggering when anywhere on the list is click (not necessarily on a link). Is also triggering when the list has no links and is clicked.

Reply:
Sorry I misunderstood this on the call - I thought you were creating the ref inside the link and attaching it to the onclick there - that's why I was surprised it was still only firing once per bulleted list.

This current implementation does only fire once per bulleted list, which is intended, but it fires if anything at all in the bulleted list is clicked, which doesn't seem intended.

I think to get both behaviours - once per list, but only if they click a link in the list, you have to create the ref in the bulleted list as you're currently doing, but pass it down as a prop so it only gets attached to the `onClick` of link elements within the bulleted list